File: Web/descubretusede/model/Seccion.php
<?php
    class Seccion {
 	public $id_seccion;
	public $id_asignatura;
	public $id_sabana;
	public $numero_seccion;
	public $jornada;
	public $profesor;
	
	
	function __construct()
	{
	}
 	
	function init($id_seccion,$id_asignatura,$id_sabana,$numero_seccion,$jornada,$profesor)
	{
		$this->id_seccion=$id_seccion;
		$this->id_asignatura=$id_asignatura;
		$this->id_sabana=$id_sabana;
		$this->numero_seccion=$numero_seccion;
		$this->jornada=$jornada;
		$this->profesor=$profesor;
	}
	
	function __destruct() 
	{
        unset($this);
	}
}
?>

File: android/app/src/main/java/cl/duoc/descubretusede/model/Seccion.java
package cl.duoc.descubretusede.model;

import java.util.ArrayList;

/**
 * Created by kurt on 13-10-2014.
 */
public class Seccion {

    private int idSeccion,numeroSeccion, idAsignatura;
    private String profesor;
    private char jornada;
    private ArrayList<Horario> horarios;

    public Seccion() {
    }

    public int getIdAsignatura() {
        return idAsignatura;
    }

    public void setIdAsignatura(int idAsignatura) {
        this.idAsignatura = idAsignatura;
    }

    public int getIdSeccion() {
        return idSeccion;
    }

    public void setIdSeccion(int idSeccion) {
        this.idSeccion = idSeccion;
    }

    public int getNumeroSeccion() {
        return numeroSeccion;
    }

    public void setNumeroSeccion(int numeroSeccion) {
        this.numeroSeccion = numeroSeccion;
    }

    public String getProfesor() {
        return profesor;
    }

    public void setProfesor(String profesor) {
        this.profesor = profesor;
    }

    public char getJornada() {
        return jornada;
    }

    public void setJornada(char jornada) {
        this.jornada = jornada;
    }

    public ArrayList<Horario> getHorarios() {
        return horarios;
    }

    public void setHorarios(ArrayList<Horario> horarios) {
        this.horarios = horarios;
    }
}
